New sample: Teams team Request Form

Katerina-Chernevskaya opened the pull request:
> We're so eager to add your sample to this repository as soon as possible, but we need all samples follow the same structure. 
> To make sure we process it as soon as possible, make sure to follow [these instructions](https://github.com/pnp/powerapps-samples/wiki/How-to-submit-a-Power-App-sample).

|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | no                               |
| New feature?    | no                               |
| New sample?     | yes                               |
| Related issues? | fixes #X, partially #Y, mentioned in #Z |

## Solution Details

|        Q                                                  |                    A                    |
| --------------------------------------------------------- | --------------------------------------- |
| Solution requires premium license                         | no                               |
| Solution uses experimental features (see [licensing guidelines](https://go.microsoft.com/fwlink/?linkid=2085130)) | no  |
| Which connectors are used?                                | SharePoint, Teams, Outlook  |
| Solution uses on-premises connectors?                     | no                               |
| Solution uses custom connectors?                          | no                               |
| Layout                                                    | Tablet, responsive                               |

## What's in this Pull Request?

The following solution components are included in this Pull Request:
* New Team Request (Canvas app)
* Create a new Team per request (Cloud flow)
Additionally the Pull Request contains the ReadMe file with detailed instructions for the instalation

Laskewitz commented:
Hi @Katerina-Chernevskaya,

Looks like a very useful sample! Could you please add connection references + environment variables to your solution, so it's easier to import the solution?

Connection references docs: https://learn.microsoft.com/en-us/power-apps/maker/data-platform/create-connection-reference
Evironment variables docs: https://learn.microsoft.com/en-us/power-apps/maker/data-platform/environmentvariables

Thanks in advance,
Daniel

Katerina-Chernevskaya commented:
Hi @Laskewitz ,

Thank you so much for the recognition and valuable advice! I updated the solution by adding 2 environment variables and 3 connection references. As well I updated Readme file - the part with installation instruction. Now the import in pretty straightforward and clear.

Thank you again for your support! :)

Kate
